README.md

```diff
@@ -146,6 +147,13 @@ select  `Go to Definition` (or press <kbd>F12</kbd>).
 
 ![Go to Definition](./assets/superbol-goto-definition.gif)
```

Comment:
One of the navigation features of vscode is "go to symbol in workspace" - is this supported and if not hard to add (#365)?

Reply:
No this is not supported yet. This apparently requires some ability to precisely scan a whole workspace, which we anticipate to be a rather costly operation, and therefore needs to be designed with care on the LSP server's side to avoid too many noticeable impact on performance.